PowerEditor/src/CMakeLists.txt

```diff
@@ -385,13 +385,13 @@ IF (WIN32)
 if ( MSVC )
 	#do not use for mingw builds
 	SET(CMAKE_CXX_FLAGS "/EHa /MP /W4")
-	SET(defs -DUNICODE -D_UNICODE -D_WIN32_WINNT=0x0600 -D_USE_64BIT_TIME_T -DTIXML_USE_STL -DTIXMLA_USE_STL -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_CRT_NON_CONFORMING_SWPRINTFS )
+	SET(defs -DUNICODE -D_UNICODE -D_WIN32_WINNT=_WIN32_WINNT_VISTA -D_USE_64BIT_TIME_T -DTIXML_USE_STL -DTIXMLA_USE_STL -DNOMINMAX -DOEMRESOURCE -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_CRT_NON_CONFORMING_SWPRINTFS -D_CRT_NON_CONFORMING_WCSTOK -D_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING )

```

Comment:
@chcg 
Just a question. Are both `-D_CRT_NON_CONFORMING_SWPRINTFS` and `-D_CRT_NON_CONFORMING_WCSTOK` needed? Notepad++ msvc builds don't use these macros anymore.

Reply:
@ozone10 Thanks. No, should be inline with the settings of the visual studio build. Corrected.